content/release-notes/2023-03-22.md

```diff
@@ -0,0 +1,9 @@
+---
+label: 'Console'
+---
+
+### What's new
```

Comment:
before this release what was consumed while on the Free Tier before the usage upgrade was also showing up as consumed on Pro tier (incorrect)
the costs were calculated correctly, but the corresponding consumption showing up in the UI was slightly higher
this was fixed in this release..